tests/HDWalletTests.cpp

```diff
@@ -21,13 +21,42 @@ namespace TW {
 TEST(HDWallet, privateKeyFromXPRV) {
     const std::string xprv = "xprv9yqEgpMG2KCjvotCxaiMkzmKJpDXz2xZi3yUe4XsURvo9DUbPySW1qRbdeDLiSxZt88hESHUhm2AAe2EqfWM9ucdQzH3xv1HoKoLDqHMK9n";
     auto privateKey = HDWallet::getPrivateKeyFromExtended(xprv, DerivationPath(TWPurposeBIP44, TWCoinTypeBitcoinCash, 0, 0, 3));
+    ASSERT_TRUE(privateKey);
```

Comment:
should we use `ASSERT_NE`?

Reply:
return value here is optional, boolean true mean it has value.